template/pyproject.toml.jinja

```diff
@@ -208,7 +208,7 @@ dependencies = [
 detached = true
 
 [tool.hatch.envs.style.scripts]
-docstrings = "pydoclint"
+docstrings = "pydoclint src/"
```

Comment:
the hatch lint step won't work within a dir associated with pydoclint ✨

Reply:
I suggest to also run pydoclint on the tests.